src/vma/sock/sock-redirect.cpp

```diff
@@ -525,10 +525,8 @@ int vma_add_conf_rule(char *config_line)
 
 	int ret = __vma_parse_config_line(config_line);
 
-#if (VMA_OPTIMIZE_LOG <= 5)
 	if (*g_p_vlogger_level >= VLOG_DEBUG)
 		__vma_print_conf_file(__instance_list);
-#endif
 
```

Comment:
As for 2: be consistence and do the same for https://github.com/Mellanox/libvma/pull/350/files#diff-ba7a8ee0726e15a62de6713b16ff9cacR126 if you see it is useful.
As for 3: it is correct. See https://github.com/Mellanox/libvma/pull/350/files/4f3b7d25e0cb98ca36c3e355e501c0efafaec152#diff-15bd24d28b25de7676c56ee4d047b293L62 as reference.